src/pages/projectSettings/tabs/GithubCommitQueueTab/GithubCommitQueueTab.tsx

```diff
@@ -100,7 +107,73 @@ export const GithubCommitQueueTab: React.VFC<TabProps> = ({
         schema={schema}
         uiSchema={uiSchema}
         disabled={isProduction() && !githubWebhooksEnabled} // TODO: Remove once EVG-16608 is fixed
+        validate={validateConflicts}
       />
     </>
   );
 };
+
+const validate = (projectType, repoData, versionControlEnabled) => (
+  formData,
+  errors
+) => {
+  const {
+    github: {
+      prTestingEnabled,
+      prTesting,
+      githubChecksEnabled,
+      githubChecks,
+      gitTagVersionsEnabled,
+      gitTags,
+    },
+    commitQueue: { enabled, patchDefinitions },
+  } = formData;
+
+  const getError = getVersionControlError(versionControlEnabled, projectType);
```

Comment:
This definition and usage is confusing me a bit. Is there anything that can be done to make it more readable?

Reply:
`getVersionControlError` is a curried function which I always find difficult to name/use clearly 😅 let me know if this helps!

Reply:
cool stuff! yes, that definitely helps, thank you!